packages/synapse-react-client/src/components/SynapseHomepage.tsx

```diff
@@ -363,67 +434,271 @@ export const SynapseHomepage: React.FunctionComponent<SynapseHomepageProps> = ({
-              url="https://adknowledgeportal.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://s3.amazonaws.com/static.synapse.org/images/BSMN.svg"
-              name="BSMN Portal"
-              description="Data and tools to study the role of brain somatic mosaicism in neuropsychiatric disease."
-              url="https://bsmn.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://s3.amazonaws.com/static.synapse.org/images/CSBC.svg"
-              name="Cancer Complexity Knowledge Portal"
-              description="Approaches, data, and tools to address important questions in basic and translational cancer research."
-              url="https://cancercomplexity.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://s3.amazonaws.com/static.synapse.org/images/dHealth.svg"
-              name="dHealth Digital Health Knowledge Portal"
-              description="Discover and download digital and mobile health data, tools, benchmarked outcomes and digital biomarkers."
-              url="https://dhealth.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://s3.amazonaws.com/static.synapse.org/images/NFPortal.svg"
-              name="NF Data Portal"
-              description="Datasets, analysis tools, resources, and publications related to neurofibromatosis and schwannomatosis."
-              url="https://nf.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://s3.amazonaws.com/static.synapse.org/images/Psychencode.svg"
-              name="Psychencode Knowledge Portal"
-              description="Investigation and large-scale data sets of genomic influences on neuropsychiatric disease."
-              url="https://psychencode.synapse.org/"
-            />
-            <SynapsePoweredPortal
-              logoUrl="https://arkportal.synapse.org/favicon.svg"
-              name="ARK Portal"
-              description="The ARK Portal hosts data generated by a network of research teams working collaboratively to deepen the understanding of Arthritis and Autoimmune and Related Diseases."
-              url="https://arkportal.synapse.org/"
-            />
-          </div>
+          <Button
+            variant="contained"
+            target="_blank"
+            href="https://sagebionetworks.jira.com/servicedesk/customer/portal/9"
+            sx={{ marginTop: '25px' }}
+          >
+            Contact us for more information
+          </Button>
+          <Link
+            target="_blank"
+            href="https://help.synapse.org/docs/FAQ.2047967233.html"
+            sx={{ display: 'block', marginTop: '25px', marginBottom: '50px' }}
+          >
+            View Frequently Asked Questions
+          </Link>
+          <Box sx={{ display: 'grid', gridTemplateColumns: '40% 20% 20% 20%' }}>
+            <p></p>
```

Comment:
Is this just a placeholder so everything else just slots nicely into the grid? If so, add simple comment (and maybe change it to a div?)